en/releases/centreon-core.md

```diff
@@ -19,6 +19,18 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 21.04.9
+
+#### Bug Fixes
+
+- [Authentication] Fixed PHP error when debug is enabled with OIDC authentication
+- [CLAPI] Fixed LDAP configuration ID for export and import
+- [Configuration] Fixed Centreon Broker configuration that could be empty or deleted
+- [Configuration] Fixed the list of host template that was not available if the database name was different from the default
+- [UX] Fixed display of tooltips in configuration forms
+- [UX] Non admin user do not have the same submenu subsections
+- [UX] Removed "Animation effects" option
+
```

Comment:
Maybe MON-11139 : [Dependency to upgrade on : centreon/centreon] - composer prod. bump nelmio/cors-bundle from 1.5.6 to 2.1.1 should appear

Reply:
no this is technical